overlay/source/main.cpp

```diff
@@ -21,16 +40,17 @@ class OverlayTest : public tsl::Overlay {
                 .storageID  = NcmStorageId_None,
             };
             rc = pmshellInitialize();
-            if (R_SUCCEEDED(rc))
+            if (R_SUCCEEDED(rc)) {
                 rc = pmshellLaunchProgram(0, &programLocation, &pid);
-            pmshellExit();
+                pmshellExit();
+            }
             if (R_FAILED(rc) || pid == 0) {
                 this->fail = rc;
                 this->msg  = "  Failed to\n"
                             "launch sysmodule";
                 return;
             }
-            svcSleepThread(100'000'000);
+            svcSleepThread(500'000'000ULL);
```

Comment:
Was this necessary?

Reply:
Surprisingly yes. In some cases, the sysmod wouldn't launch in time so the overlay would say it failed. The number is chosen was random. Maybe it could be reduced further.

I'll make more tests tomorrow because *maybe" the slow startup was causes by some previous changes I had made.